content/docs/next-release/spm.md

```diff
@@ -83,9 +96,12 @@ graph
             PROMETHEUS_EXPORTER
         end
     end
+
     style UI fill:#9AEBFE,color:black
     style QUERY fill:#9AEBFE,color:black
+    style COLLECTOR fill:#9AEBFE,color:black
 
+    style OTLP_EXPORTER fill:#404ca8,color:white
```

Comment:
Since OTLP exporter is part of the OTEL project, I thought to keep these colours consistent.

Reply:
Service outer box could be different color

Reply:
Yup, I agree, thanks. Updated to a grey color, grouped them in order of appearance and capitalized the hex codes for consistency in: [13e3e2a](https://github.com/jaegertracing/documentation/pull/617/commits/13e3e2a07d41d6a630f6bb39ba27f3e7188f99cd)

Reply:
I've also updated the rendered image in the PR description. Let me know if you prefer a different color.